taf/exceptions.py

```diff
@@ -88,15 +88,19 @@ def __init__(self, repo_path: str, message: str):
 
 
 class RepositoryNotCleanError(TAFError):
-    def __init__(self, repo_name: str):
-        message = f"Repository {repo_name} has uncommitted changes. Commit and push or revert the changes and run the command again."
+    def __init__(self, repo_names: List[str]):
+        repo_list = ", ".join([repo for repo in repo_names])
```

Comment:
I don't think that you need to use list comprehension here because repo_namea is already a list